frontend/src/app/components/backlog-content/backlog-content.component.ts

```diff
@@ -205,7 +206,26 @@ export class BacklogContentComponent implements OnInit {
     });
   }
 
+  ngOnChanges(): void {
+    this.filterItems();
+  }
+
   ngOnInit(): void {
+    if (this.currentUser === undefined) {
```

Comment:
It will allow you to simplify this part of code, moreover I think we should not let user access to this component if current user is undefined